Formula/ponyc.rb

```diff
@@ -12,23 +12,21 @@ class Ponyc < Formula
     sha256 "216889f9856cfb4da8bb46a9aaac3af38e4f1f8f349b5928645ea0d266231b8b" => :high_sierra
   end
 
-  # https://github.com/ponylang/ponyc/issues/1274
-  # https://github.com/Homebrew/homebrew-core/issues/5346
+  # Clang cannot find system headers if Xcode CLT is not installed
   pour_bottle? do
-    reason <<~EOS
-      The bottle requires Xcode/CLT 8.0 or later to work properly.
-    EOS
-    satisfy { DevelopmentTools.clang_build_version >= 800 }
+    reason "The bottle needs the Xcode CLT to be installed."
+    satisfy { MacOS::CLT.installed? }
   end
```

Comment:
I'm not completely sure what to do with this block, I've copied it from `llvm` formula

Reply:
The reason for that will have been because of the default sysroot. Is there one here?

What gets build here? The entirety of the LLVM project including all subprojects?

Reply:
> The reason for that will have been because of the default sysroot. Is there one here?

I don't see anything like this. Probably we could remove it.

> What gets built here? The entirety of the LLVM project including all subprojects? The bottle output size is tiny for that.

The only things related to llvm build are: 
https://github.com/ponylang/ponyc/blob/0.35.1/lib/CMakeLists.txt#L88-L100